alpine-nix-rails-nginx/etc/nginx/nginx.conf

```diff
@@ -63,6 +62,12 @@ http {
   proxy_set_header       X-Request-Start   "t=${msec}";
   proxy_set_header       Connection        "";
 
+  proxy_temp_path       /var/cache/nginx/proxy_temp;
+  client_body_temp_path /var/cache/nginx/client_body_temp;
+  fastcgi_temp_path     /var/cache/nginx/fastcgi_temp;
+  uwsgi_temp_path       /var/cache/nginx/uswgi_temp;
+  scgi_temp_path        /var/cache/nginx/scgi_temp;
```

Comment:
When I get some time I am going to submit a patch upstream to `nixpkgs` that makes `fastcgi`, `uswgi` and `scgi` optional (among others). My preference is to have a minimal nginx compilation, so I'll work on making some of those other things optional.